### ISSUE TITLE
Error on writes, OK on reads

### ISSUE THREAD

**Issue by treeder** [eeonevision/kucoin-go#5]
I have a read/write token I created on KuCoin and I'm able to read things, but I can't create an order, it returns this:

```
 {"msg":"Signature verification failed","timestamp":1514554308458,"code":"UNAUTH","success":false}
```

Anyone else getting that?

**Comment by eeonevision** [comment]
Hi! Can you debug the strForSign value?
It places at: https://github.com/eeonevision/kucoin-go/blob/1a6333d86f831a2e8668d0e6cc69a1a5d851f568/client.go#L147

Send to this issue the value and stack trace to this var.

**Comment by eeonevision** [comment]
Fixed in #6
